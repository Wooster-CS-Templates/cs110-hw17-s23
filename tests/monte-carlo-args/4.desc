Run command: ./monte-carlo 6 2 100 1000

