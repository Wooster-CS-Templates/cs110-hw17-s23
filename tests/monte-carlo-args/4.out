Usage: ./monte-carlo <sides per die> <dice count> <roll count>
